src/website/views/custom-elements/alert.pug

```diff
@@ -0,0 +1,189 @@
+---
+title: Alert
+scripts:
+  - js/ce/ux-chi-ce
+---
+p.-text Alert messages provide contextual feedback for user actions.
+
+h3 Examples
+p.-text
+  | To render an alert, use the tag <code>&lt;chi-alert&gt;</code>.
+  | A value (e.g. <code>warning</code>) must be set to attribute <code>color</code> so as to provide proper styling.
+
+h4 Base
+.example.-mb--3
+  .-p--1
+    each type in ['success', 'danger', 'warning', 'info', 'muted']
+      chi-alert.-m--2(color=type)= `This is a ${type} alert`
+  :code(lang="html")
+    <chi-alert color="success" >This is a success alert</chi-alert>
```

Comment:
Please, remove extra space between last property and _>_, and fix in further occurrences.